src/Integration.UnitTests/Connection/ConnectionWorkflowTests.cs

```diff
@@ -65,14 +64,7 @@ public void TestInit()
             this.host = new ConfigurableHost(this.serviceProvider, Dispatcher.CurrentDispatcher);
             this.host.SetActiveSection(ConfigurableSectionController.CreateDefault());
             this.host.SonarQubeService = this.sonarQubeServiceMock.Object;
-            this.projectSystemHelper = new ConfigurableVsProjectSystemHelper(this.serviceProvider);
-
-            this.sonarQubeServiceMock.Setup(x => x.GetAllPluginsAsync(It.IsAny<CancellationToken>()))
-                .ReturnsAsync(new List<SonarQubePlugin>
-                {
-                    new SonarQubePlugin(MinimumSupportedSonarQubePlugin.CSharp.Key, MinimumSupportedSonarQubePlugin.CSharp.MinimumVersion),
-                    new SonarQubePlugin(MinimumSupportedSonarQubePlugin.VbNet.Key, MinimumSupportedSonarQubePlugin.VbNet.MinimumVersion)
-                });
```

Comment:
The product code no longer fetches the list of plugins from the server so the test setup is simpler.